### DISCUSSION TITLE
I love the integration - any chance it's going to HA Core as well?

### DISCUSSION THREAD

**Discussion by scotrod** [petretiandrea/home-assistant-tapo-p100#630] [Q&A] [answered]
Hey. Lovin' the integration, I just want to better understand why are some integrations (like the ones in the HACS store) not in HA Core? Are the conditions to enter there so rough? Do you need to provide some sort of extensive support/security checks? I know that this is a general question, but this is one integration that I hope one day will take its place in the official integrations store in HA.

Thanks!

**Comment by petretiandrea** [comment, marked as answer]
Thanks for support, to add integration to core I must provide more testing codebase, but actually I don't have enough time to do that, but I'm working on it ;)
